src/Playwright/Core/BindingCall.cs

```diff
@@ -42,15 +42,18 @@ internal async Task CallAsync(Delegate binding)
                     },
                 };
 
-                if (methodParams.Length == 1 && methodParams[0] == typeof(IJSHandle))
+                if (_initializer != null)
```

Comment:
It is never null, the call sites above are wrong.

Reply:
want me to fix the call sites in a diff pr?

Reply:
Yes. Although I barely keep the pace approving those.